microshift_updating/microshift-update-options.adoc

```diff
@@ -0,0 +1,79 @@
+:_content-type: ASSEMBLY
+[id="microshift-update-options"]
+= Update options with {product-title} and {op-system-bundle}
+include::_attributes/attributes-microshift.adoc[]
+:context: microshift-update-options
+
+toc::[]
+
+You can update {op-system-ostree-first} or {op-system-base-full} with or without updating {product-title-first} as long as the two versions are compatible. See the following table for details:
+
+include::snippets/microshift-rhde-compatibility-table-snip.adoc[leveloffset=+1]
+
+*{product-title} update paths*
+
+* Generally Available Version 4.14.0 to 4.14.z on {op-system-ostree} 9.2
+* Generally Available Version 4.14.0 to 4.14.z on {op-system} 9
+
+[IMPORTANT]
+====
+Updates of {product-title} from one minor version to the next must be in sequence. For example, you cannot update from 4.14 to 4.16. You must update 4.14 to 4.15.
+====
+
+[id="microshift-update-options-standalone-updates_{context}"]
+== Standalone {product-title} updates
+
+You can update {product-title} without reinstalling the applications you created. {op-system} or {op-system-ostree} updates are also not required to update {product-title}, as long as the existing operating system is compatible with the new version of {product-title} that you want to use.
+
+{product-title} operates as an in-place update and does not require removal of the previous version. Data backups beyond those required for the usual functioning of your applications are also not required.
+
+[id="microshift-update-options-rpm-ostree-updates"]
+=== RPM-OSTree updates
+You can update {product-title} on an `rpm-ostree` system such as {op-system-ostree} by building a new image containing the new version of {product-title}. Ensure that version of the image you want to build is compatible with the new version of {product-title} that you want to use.
+
+The following features are available in the {op-system-ostree} update path:
+
+* The system automatically rolls back to a previous healthy system state if the update fails.
+* Back ups and restores of data are automatic.
+* Applications do not need to be reinstalled.
+* You can update an application without updating {product-title} using this update type.
+* Storage provisioning only needs to be updated if the new image of {op-system-ostree} that you create requires new provisioning.
```

Comment:
Why call out storage and networking configuration specially here? All updates only need to include what needs to be changed.